### PR TITLE
[ADD] payment: added twint to mollie

### PR DESCRIPTION
**Pull request by jeep-odoo**
Mollie has now started supporting Twint as a payment method.

task-3859061

### CODE REVIEW

File: addons/payment_mollie/const.py [hunk 1]
```diff
@@ -66,4 +66,5 @@
     'bank_transfer': 'banktransfer',
     'p24': 'przelewy24',
     'sepa_direct_debit': 'directdebit',
+    'twint': 'twint',
```
Comment (kcv-odoo) [inline addons/payment_mollie/const.py:69]: Do we really need to add this mapping if pm code and code from mollie side is same? :thinking:

File: addons/payment/data/payment_provider_data.xml [hunk 2]
```diff
@@ -271,6 +271,7 @@
                          ref('payment.payment_method_paysafecard'),
                          ref('payment.payment_method_p24'),
                          ref('payment.payment_method_sofort'),
+                         ref('payment.payment_method_twint'),
```
Comment (kcv-odoo) [inline addons/payment/data/payment_provider_data.xml:274]: Need upgrade script for that change  :zombie:

**Review by kcv-odoo — CHANGES_REQUESTED** [review]
Thank you for your work :+1: 
Added two comments!

File: addons/payment_mollie/const.py (hunk 1 above)
Comment (jeep-odoo) [reply addons/payment_mollie/const.py]: Oh yes, it's working without this too. Thanks for pointing it out :+1:

**Review by kcv-odoo — APPROVED** [review]
LGTM :100: 

About commit title:
Your header should complete this sentence `if applied, this commit will <header>`
so it should be `add` not added

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

Don't forget to backport in 17.0 :)

**Pull request closed without merging**